src/Symfony/Component/DomCrawler/Test/Constraint/CrawlerSelectorCount.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DomCrawler\Test\Constraint;
+
+use PHPUnit\Framework\Constraint\Constraint;
+use Symfony\Component\DomCrawler\Crawler;
+
+final class CrawlerSelectorCount extends Constraint
+{
+    public function __construct(private readonly string $selector, private readonly int $count)
+    {
+    }
+
+    public function toString(): string
+    {
+        return sprintf('selector "%s" count is "%d"', $this->selector, $this->count);
+    }
+
+    /**
+     * @param Crawler $crawler
+     */
+    protected function matches($crawler): bool
+    {
+        return $this->count === \count($crawler->filter($this->selector));
+    }
+
+    /**
+     * @param Crawler $crawler
+     */
+    protected function failureDescription($crawler): string
+    {
+        return sprintf('the Crawler selector "%s" was expected to be found %d time(s) but was found %d time(s)', $this->selector, $this->count, \count($crawler->filter($this->selector)));
```

Comment:
Does other constraint failure and toString messages also start with a lowercase letter? If yes, just resolve this comment, thanks

Reply:
yes I have checked other files in Constraint folder so I wanted to stay coherent with the rest and keep it lowercase also.